.github/workflows/ios.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches: [master, staging, production, next, react-native]
 
+  workflow_dispatch:
```

Comment:
what this?

Reply:
allows manual trigger of the build flow. you added it to android, it was missing for ios